mail_activity_board/models/mail_activity_mixin.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2016 David Juaneda - <djuaneda@sdi.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import models
+
+
+class MailActivityMixin(models.AbstractModel):
+    _inherit = 'mail.activity.mixin'
+
+    def redirect_to_activities(self, **kwargs):
+        """
+            Redirects to the list of activities of the object shown.
+
+            Redirects to the activity board and configures the domain so that
+            only those activities that are related to the object shown are
+            displayed.
+
+            Add to the title of the view the name the class of the object from
+            which the activities will be displayed.
+
+            :param kwargs: Dictionary that contains the id of the object
+                            and the model it is about.
+            :return: action.
```

Comment:
Reduce 4 spaces of indentation in all this docstring.

FTR https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
Thanks for the link.